projects/packages/forms/src/dashboard/index.js

```diff
@@ -1,8 +1,15 @@
 import { render } from '@wordpress/element';
+import { Provider } from 'react-redux';
 import Inbox from './inbox';
+import store from './state';
 
 window.addEventListener( 'load', () => {
 	const container = document.getElementById( 'jp-forms-dashboard' );
 
-	render( <Inbox />, container );
+	render(
+		<Provider store={ store }>
+			<Inbox />
+		</Provider>,
+		container
+	);
```

Comment:
Same as above, could we switch to `@wordpress/data` for all of this?

Here is an example, if you're looking for inspiration:
https://github.com/Automattic/jetpack/tree/trunk/projects/packages/videopress/src/client/state